### DISCUSSION TITLE
Is there an API to access via react-js?

### DISCUSSION THREAD

**Discussion by agbozo1** [GoogleChrome/lighthouse#13041] [Q&A] [answered]
I want to know if its possible to make requests to lighthouse via an API, and receive json response on reactJS.

**Comment by patrickhulce** [comment, marked as answer]
I think you are interested in the [PageSpeed Insights API](https://developers.google.com/speed/docs/insights/v5/get-started). 

The npm library cannot be run client-side in a standard webapp because it requires a privileged connection to the debugging protocol of Chromium.

**Reply by agbozo1** [reply]
@patrickhulce thank you for this. That is exactly what I have been looking for.
Kind Regards
